corehq/toggles.py

```diff
@@ -849,6 +849,16 @@ def _ensure_valid_randomness(randomness):
     """
 )
 
+USH_WEB_USER_CASE_CREATION = StaticToggle(
+    'web_user_case_creation',
+    "USH: Enable the creation of user cases for web users.",
+    TAG_CUSTOM,
+    help_link='.',
```

Comment:
This PR looks almost ready, so it'd be a good time to create docs and add the link here. You can make a child page of [this page](https://confluence.dimagi.com/pages/viewpage.action?pageId=41485226). It doesn't need to be long - [this page](https://confluence.dimagi.com/display/saas/USH%3A+Enable+Web+User+Activity+Report) is a basic example of docs for another feature flag that's USH-owned and still under development, like your new flag.